samples/OneClickDemo/OneClickDemo-Katalog.sh

```diff
@@ -6,7 +6,7 @@ set -e
 HELM_VERSION=v3.10.3
 YQ_VERSION=4.6.0
 KUBE_VERSION=1.22.0
-KIND_VERSION=0.14.0
+KIND_VERSION=0.17.0
```

Comment:
can you add a rule to the main Mainfile to automatically update these versions?

Reply:
hmmm
I don't see a way to automatically update these versions from the Makefile, as when you use OneClickDemo you don't clone the Fybrik repo or use its Makefile, only use `curl` to a script in Fybrik main Github repo, not locally...

I can add a `curl` command inside the script that reads the current updated version from Fybrik master branch if you think that is better

Reply:
No, I thought about a templatefile(s) that can be "compiled" into the final file that will be used by curl. 
We can talk tomorrow.

Reply:
as we discussed, added a make target 'reconcile-requirements'